hypervisor/arch/x86/guest/vlapic.c

```diff
@@ -1240,6 +1263,19 @@ vlapic_icrlo_write_handler(struct acrn_vlapic *vlapic)
 	return 0;	/* handled completely in the kernel */
 }
 
+/**
+ * @brief Get pending virtual interrupts for vLAPIC.
+ *
+ * @param vlapic[in] Pointer to target vLAPIC data structure
+ * @param vector[inout] Pointer to vector buffer and will be filled
```

Comment:
```suggestion
 * @param vecptr[inout] Pointer to vector buffer and will be filled
```

Reply:
@gvancuts  thanks for comments ,will update